<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneCognitionDialog</class>
 <widget class="QDialog" name="SceneCognitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>场景认知能力评估</string>
  </property>
  <widget class="QLabel" name="cognitiveTestImageLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>测试图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonA">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonB">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonC">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
